pkg/cli/testdata/MockClient.go

```diff
@@ -3,6 +3,7 @@ package testdata
 import (
 	"context"
 	"fmt"
+	"github.com/acorn-io/acorn/pkg/labels"
```

Comment:
hm. goimports lint check should have caught this. can you rebase this pr to see if that will cause the linter to catch it?